articles/active-directory/develop/tutorial-v2-asp-webapp.md

```diff
@@ -391,6 +391,7 @@ To register your application and add the app's registration information to your
 
 1. Replace `ClientId` with the Application ID you just registered.
 1. Replace `redirectUri` with the SSL URL of your project.
+1. If you plan to use Microsoft Accounts (MSA) to authenticate, you should replace `Tenant` with "consumer" 
```

Comment:
```suggestion
1. If you plan to use only Microsoft Accounts (MSA) to authenticate, you should replace `Tenant` with `consumers` 
```

Reply:
> cc: @mmacy to decide if we take this as there could be an impact on all quickstarts

Several of them do (list below), but instead of doing this in the quickstarts or tutorials themselves, it should appear in an app registration tutorial and should be pulled in via an INCLUDE so we only modify it once to update everywhere.

Current mentions of this in quickstarts and tutorials:

```console
quickstart-v2-angular.md
quickstart-v2-dotnet-native-aspnet.md
quickstart-v2-javascript-auth-code-angular.md
quickstart-v2-javascript-auth-code-react.md
quickstart-v2-javascript-auth-code.md
quickstart-v2-javascript.md
tutorial-v2-javascript-auth-code.md
tutorial-v2-nodejs-console.md
tutorial-v2-nodejs-desktop.md
tutorial-v2-nodejs-webapp-msal.md
```